### ISSUE TITLE
Tongue claudication

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11450]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

PMID: 30385998 Jaw claudication occurs in about 40% of patients and again is usually early finding but tongue claudication is rare
PMID: 37021621 In a study of temporal artery biopsies, the symptom correlating with a positive biopsy most frequently was jaw claudication and was one of two symptoms heightening the pre test probability of a positive temporal artery biopsy in a large metaanalysis. Similar findings that can be seen are tongue and swallowing claudication as well as throat pain.
PMID: 36344353 Even though GCA-related headache is traditionally thought to be located over the temples, it can also be frontal, occipital, or generalized. Involvement of cranial arteries can also cause scalp tenderness, dysphagia, and jaw and tongue claudication.

**Parent term (use hpo.jax.org/app)**
Movement abnormality of the tongue  HP:0000182

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell arteritis ORPHA:397

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6001463
- label: Tongue claudication
- parents: [Intermittent claudication (HP:0004417)](https://hpo.jax.org/app/browse/term/HP:0004417)
- pmid: PMID:30385998
- def: Pain in the tongue induced by speaking or chewing.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11450
- synonyms: Lingual claudication(NONE)
